tezos/context/src/lib.rs

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) SimpleStaking and Tezedge Contributors
 // SPDX-License-Identifier: MIT
-#![forbid(unsafe_code)]
+
+// TODO: re-enable this once ocaml-interop has been upgraded
+//#![forbid(unsafe_code)]
```

Comment:
@tizoc 
what about this?

Reply:
@bkontur the ocaml-interop-upgrade branch re-enables it. Remember that originally this branch depended on the other, but things had to be turned around. That required some of the newer ocaml-interop features to be imported directly into that branch, and that code requires `unsafe` features. The ocaml-interop upgrade branch removes the imported code and re-enables forbid-unsafe